test/benchee/utility/file_creation_integration_test.exs

```diff
@@ -41,6 +43,7 @@ defmodule Benchee.Utility.FileCreationIntegrationTest do
         assert output =~ @file_name_2
       after
         File.rm_rf! @directory
+        File.rm_rf! "testing"
```

Comment:
Because we were doing `rm_rf!` on `testing/files`, we were always leaving the `testing` folder around, which has bothered me for a bit. Now that's gone, too!